Sources/TDS/Utilities/TDS+GeneralRules.swift

```diff
@@ -46,12 +49,17 @@ public typealias Precision = UInt8
 /// SCALE
 public typealias Scale = UInt8
 
-// TODO: GEN_NULL
-// TODO: CHARBIN_NULL
-// TODO: FRESERVEDBIT
-// TODO: FRESERVEDBYTE
+/// GEN_NULL
+public typealias GenNull = UInt8
 
-/// UNICODECHAR
-public typealias UnicodeChar = UInt16
+/// CHARBIN_NULL
+public typealias CharBinNull = UInt16
 
+/// FRESERVEDBIT
+public typealias FReservedBit = TDS.Bit
 
+/// FRESERVEDBYTE
+public typealias FReservedByte = TDS.Byte
```

Comment:
For now lets leave this unscoped as it doesnt need specification in this context.